app/models/item.rb

```diff
@@ -14,4 +14,36 @@ def self.order_by(sorted)
 			all
 		end
 	end
+
+	def self.search_by_params(param)
+		if param[:name] && (param[:min_price] || param[:max_price])
```

Comment:
Thorough coverage with conditionals, this looks great and I'm looking forward to build off of this work.